dataprofiler/tests/profilers/test_numeric_stats_mixin_profile.py

```diff
@@ -584,7 +584,7 @@ def test_diff(self):
             'max': [4, None],
             'sum': 'unchanged',
             'mean': 0.3,
-            'variance': -8.362573099415204,
```

Comment:
why the reduction?

Reply:
also i believe the exact value is `(9*20/19) - (1*10/9)`

Reply:
no need to do almost equal

Reply:
I was getting issues with Torch on this number due to the decimals, but this looks like a good workaround.